Add cookie implementation details for the NPM package (#7051)
segmentio/segment-docs: src/connections/sources/catalog/libraries/website/javascript/cookie-validity-update.md

  }
})
```

> To set these values via the [NPM package](https://github.com/segmentio/analytics-next/tree/master/packages/browser), you can set these cookie values like so:

Reviewer: ```suggestion
To set the values using the [NPM package](https://github.com/segmentio/analytics-next/tree/master/packages/browser){:target="_blank"}, set the cookie values like to:
```
